### PR TITLE
Updating the high level contract docs.

### PR DESCRIPTION
**Pull request by scottynomad**

### CODE REVIEW

**Review by amiecorso — APPROVED** [review]

File: contracts/README.md [hunk 1]
```diff
@@ -1,8 +1,28 @@
 # Nori Smart Contracts
 
+Nori's product is the NRT or Nori _Removal_ Tonne which can be thought of
+as a carbon credit granted to a supplier for the CO2 they have removed from
+the atmosphere.
+The supplier consigns this asset in Nori's marketplace and gets paid in _NORI_ token.
+A buyer of NRTs is issued a _Certificate_
```
Comment (rlfarman) [inline contracts/README.md:7]: Weird formatting here?

File: contracts/README.md [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+(NCCR) which is non-transferrable and owns the NRTs that were swapped from
```
Comment (rlfarman) [inline contracts/README.md:8]: Could be a little clearer here - does the Certificate own the NRTs, or does the buyer own the NRTs? If the buyer owns the NRTs, can they resell them?

File: contracts/README.md [hunk 3]
```diff
@@ -1,8 +1,28 @@
...
+the supplier for NORI. If a supplier is found to have prematurely released the
```
Comment (rlfarman) [inline contracts/README.md:9]: I would almost put this section about releasing/the insurance reserve beneath the block below, so people have a better idea of what the insurance reserve is/how it funds replacement NRTs with NORI.

**Review by rlfarman — APPROVED** [review]
Couple minor comments but good overview!

**Pull request merged**